DZ_12/task_3.py

```diff
@@ -0,0 +1,32 @@
+
+# 3. Переписати декоратор із першого завдання,
+# щоб він приймав цілочисельний аргумент `times`.
+# Стільки разів виконувавати друк назви функції і часу, скільки ‘times’ задано.
+
+import time
+
+def func_decorator(func):
+    def wrapper(*args, **kwargs):
+        for i in range(3):   # я тільки додала цикл for з times = 3
+            print(f"This function: {func.__name__} was launched at {time.strftime('%Y-%m-%d %H:%M:%S')} ")
+        return func(*args, **kwargs)
+
+    return wrapper
+
+
+data = [6745, 98768, 79847, 98946, 6787, 8467, 86678]
+
+@func_decorator
```

Comment:
the task was to create decorator with parameter like
```suggestion
@func_decorator(times=2)
```